<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Crésus</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QWidget" name="m_button_window" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="m_retrait_but">
         <property name="text">
          <string>Ajouter retrait</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_monnaie_but">
         <property name="text">
          <string>Editer porte-monnaie</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::VLine</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QWidget" name="m_account_window" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="m_bilan_tab">
          <attribute name="title">
           <string>Bilan</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_17">
           <item>
            <widget class="AccountWidget" name="m_user_info" native="true"/>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string>Capitaux</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QGroupBox" name="m_capital_liquid">
                <property name="title">
                 <string>Capitaux Liquides</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_8">
                 <item>
                  <widget class="AccountWidget" name="m_cash_amount" native="true"/>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="m_capital_positif">
                <property name="title">
                 <string>Capitaux Positifs</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_9"/>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="m_capital_negatif">
                <property name="title">
                 <string>Capitaux Négatifs</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_10"/>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="m_debt_group_box">
             <property name="title">
              <string>Dettes</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QGroupBox" name="m_debt_short">
                <property name="title">
                 <string>Dettes Court-termes</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_11"/>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="m_debt_long">
                <property name="title">
                 <string>Dettes Long-termes</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_12"/>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Expanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="m_depense_tab">
          <attribute name="title">
           <string>Dépenses</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="1" column="0">
            <widget class="QGroupBox" name="groupBox_3">
             <property name="title">
              <string>Dépenses</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_13">
              <item>
               <widget class="QGroupBox" name="m_personal_expenses">
                <property name="title">
                 <string>Personnelles</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_14">
                 <item>
                  <widget class="MonthlyTableWidget" name="m_groceries" native="true"/>
                 </item>
                 <item>
                  <widget class="MonthlyTableWidget" name="m_clothing" native="true"/>
                 </item>
                 <item>
                  <widget class="MonthlyTableWidget" name="m_hygiene" native="true"/>
                 </item>
                 <item>
                  <widget class="MonthlyTableWidget" name="m_health" native="true"/>
                 </item>
                 <item>
                  <widget class="MonthlyTableWidget" name="m_phone" native="true"/>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="m_housing_expenses">
                <property name="title">
                 <string>Logement</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_15"/>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="m_transport_expenses">
                <property name="title">
                 <string>Transport</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_16"/>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>Revenus</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QGroupBox" name="m_active_revenues">
                <property name="title">
                 <string>Revenus Actifs</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_7">
                 <item>
                  <widget class="MonthlyTableWidget" name="m_net_salary" native="true"/>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="m_passive_revenues">
                <property name="title">
                 <string>Revenus Passif</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_6">
                 <item>
                  <widget class="MonthlyTableWidget" name="m_interests" native="true"/>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="2" column="0">
            <spacer name="verticalSpacer_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="m_quit_action"/>
   </widget>
   <widget class="QMenu" name="menu_dition">
    <property name="title">
     <string>Édition</string>
    </property>
    <addaction name="m_retrait_action"/>
    <addaction name="m_monnaie_action"/>
    <addaction name="separator"/>
    <addaction name="m_pref_action"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>Aide</string>
    </property>
    <addaction name="m_about_action"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menu_dition"/>
   <addaction name="menuAide"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="m_gain_action"/>
   <addaction name="m_depense_action"/>
   <addaction name="m_compte_action"/>
  </widget>
  <action name="m_open_action">
   <property name="text">
    <string>Ouvrir</string>
   </property>
  </action>
  <action name="m_save_action">
   <property name="text">
    <string>Sauvegarder</string>
   </property>
  </action>
  <action name="m_quit_action">
   <property name="text">
    <string>Quitter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="m_depense_action">
   <property name="text">
    <string>Ajouter une Dépense</string>
   </property>
  </action>
  <action name="m_retrait_action">
   <property name="text">
    <string>Ajouter un retrait</string>
   </property>
  </action>
  <action name="m_gain_action">
   <property name="text">
    <string>Ajouter un Gain</string>
   </property>
  </action>
  <action name="m_monnaie_action">
   <property name="text">
    <string>Éditer porte-feuille</string>
   </property>
  </action>
  <action name="m_about_action">
   <property name="text">
    <string>À propos</string>
   </property>
  </action>
  <action name="m_pref_action">
   <property name="text">
    <string>Préférences</string>
   </property>
  </action>
  <action name="m_compte_action">
   <property name="text">
    <string>Ajouter un Compte</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AccountWidget</class>
   <extends>QWidget</extends>
   <header>interfacewidgets.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MonthlyTableWidget</class>
   <extends>QWidget</extends>
   <header>interfacewidgets.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>m_quit_action</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_about_action</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>about()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_depense_action</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>ajouterDepense()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_gain_action</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>ajouterGain()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_monnaie_action</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>editWallet()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_open_action</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>openFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_retrait_action</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>ajouterRetrait()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_save_action</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_monnaie_but</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>editWallet()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_retrait_but</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ajouterRetrait()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pref_action</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>editPreferences()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_compte_action</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>ajouterCompte()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>openFile()</slot>
  <slot>saveFile()</slot>
  <slot>about()</slot>
  <slot>ajouterDepense()</slot>
  <slot>ajouterRetrait()</slot>
  <slot>ajouterGain()</slot>
  <slot>editWallet()</slot>
  <slot>ajouterCompte()</slot>
  <slot>editPreferences()</slot>
 </slots>
</ui>
